app/helpers/add-line-breaks.js

```diff
@@ -0,0 +1,9 @@
+import Ember from 'ember';
+
+export function addLineBreaks(params/*, hash*/) {
```

Comment:
Is the commented out `hash` intentionally left here?

Reply:
Yeah it was. The generator for helpers scaffolds out these two parameters, but my helper implementation only uses the first one. 

I commented out the second because it would otherwise be an unused variable, but left it in a comment to suggest it was available. I suppose there's not much value in leaving the comment there.